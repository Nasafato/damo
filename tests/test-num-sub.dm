def main() : int {
	// Variable declarations come first
	num i;
	num j;

	// Statements come second
	j = 1.7;
	i = .4;
	print_num(i);
	print_num(j);
	print_num(j-i);
	return 0;
}
