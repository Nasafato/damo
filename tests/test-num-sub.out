0.400000
1.700000
1.300000